src/drivers/code_hosting/gitlab.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+
+function create_pull_request {
+  local repository=$1
+  local src="merge_request%5Bsource_branch%5D=${2}"
+  local dest="merge_request%5Btarget_branch%5D=${3:-master}"
```

Comment:
Then I believe master needs to become `$MAIN_BRANCH_NAME` since git town allows you to configure what you main development branch.